lib/typedeclarations.d.ts

```diff
@@ -78,10 +78,22 @@ declare module "zookeeperWithCallbacks" {
         static get ZOO_LOG_LEVEL_INFO(): number;
         /** @deprecated @returns {number} 4 */
         static get ZOO_LOG_LEVEL_DEBUG(): number;
+        /** @deprecated @returns {number} 0 */
```

Comment:
(see the feedback about deprecated features [here](https://github.com/yfinkelstein/node-zookeeper/pull/255/files#r548971650)